nf_core/lint.py

```diff
@@ -335,14 +366,58 @@ def check_version_consistency(self):
         self.passed.append((7, "Version tags are numeric and consistent between container, release tag and config."))
 
 
+    def check_conda_env_yaml(self):
+        """ Check the conda environment file - that versions are pinned and are the latest version """
```

Comment:
Perhaps more like:

"Check the conda environment file.  The check will fail if the package dependency version is not pinned and returns a warning, if version is not the latest available on https://anaconda.org/."